tests/app/authentication/test_user_id_generator.py

```diff
@@ -0,0 +1,44 @@
+from app.authentication.user_id_generator import UserIDGenerator
+from app.metadata.metadata_store import MetaDataConstants
+from app.authentication.invalid_token_exception import InvalidTokenException
+from app import settings
+import unittest
+
+
+class TestUserIDGenerator(unittest.TestCase):
+
+    def setUp(self):
+        settings.EQ_SERVER_SIDE_STORAGE = True
+
+    def test_generate_id(self):
+        user_id_1 = UserIDGenerator.generate_id(self.create_token('1', '2', '3'))
+        user_id_2 = UserIDGenerator.generate_id(self.create_token('1', '2', '3'))
+        user_id_3 = UserIDGenerator.generate_id(self.create_token('1', '2', '4'))
```

Comment:
Still needs (2, 2, 4) != user_id_3 and
and (1,3,3) != user_id_2 to cover each element changing on its own....